src/controller/cve.controller/index.js

```diff
@@ -81,25 +68,22 @@ router.get('/cve',
   /*
   #swagger.tags = ['CVE Record']
   #swagger.operationId = 'cveGetFiltered'
-  #swagger.summary = "Retrieves all CVE records (accessible to all users)"
+  #swagger.summary = "Retrieves all CVE entries (accessible to all users)"
```

Comment:
We should add 'after applying the query params as filters' to align the description with the GET /cve-id endpoint